### ISSUE TITLE
InvalidStateError and No memory map available

### ISSUE THREAD

**Issue by NodeNXT** [Omega-Numworks/Online-Installer#3]
The [https://omega-numworks.github.io/installer/omega/n0100/](https://omega-numworks.github.io/installer/omega/n0100/) doesn't work. My calculator is detected but the buttons do nothing.

**Comment by quentinguidee** [comment]
Hello! Thank you for trying to install **Omega**.

We are sorry that the installer does not work for you.
Can you tell us if you get a particular error during the installation (or at what stage it does not work), and if you are sure to have the model n0100 and not the model n0110 (which will be supported soon)?

Thank you!

**Comment by NodeNXT** [comment]
Hello !

Yes I have the N0100 model.

I selected my calculator with the first button "Connect to NumWorks calculator" and I select my calculator : I have that error :  `InvalidStateError: The device must have a configuration selected.`

Do you speak french ?

**Comment by quentinguidee** [comment]
Yes I speak french but I avoid to speak french here since we have english contributors... If french is easier for you, we have a french channel on Slack [here](https://join.slack.com/t/omeganumworks/shared_invite/enQtNzg3NDI0NDkxMzQ1LTgzZTYxNjI3NGEwMGFhYzdiYTEwYzk5ZWYwMGVlOWM0YjhlNTJkODVjZjllZWIzNmQ0YjY0MmQ4ZjVjNDc1ZjA).

As for the problem, I have it too, and I solve it by refreshing the page when I receive the error. The calculator is still connected after refreshing, and the flash button is operational.

I will keep this issue open until it is resolved. Tell me if my workaround works for you!

**Edit:** This installer is still in beta, so don't hesitate to report bugs!

**Comment by NodeNXT** [comment]
Okay

I tried to refresh but it doesn't work, my calculator is not detected automatically after the refresh and I have the same error when I select the calculator manually.

**Comment by NodeNXT** [comment]
I succeeded ! :p But I have a new error : `No memory map available` when I click on "Flash".

**Comment by quentinguidee** [comment]
I just tried and also have the No memory map error :laughing: @RedGl0w is working on it

**Comment by RedGl0w** [comment]
When you disconnect and connect again, does it persist ?

**Comment by NodeNXT** [comment]
No, when I disconnect and reconnect, it's working

**Comment by RedGl0w** [comment]
Ok
My calculator works and yours don't so it's a bit strange...
I'm looking for a solution

**Comment by NodeNXT** [comment]
It is working on my calculator now

**Comment by RedGl0w** [comment]
Ok
It's a little bit weird
Did you reload the page ?

**Comment by NodeNXT** [comment]
No, I have just clicked on connect then I clicked on "use Omega" and "Flash" without refresh the page. It's really strange xD

**Comment by RedGl0w** [comment]
Yes it is
